modules/ROOT/pages/index-appliance.adoc

```diff
@@ -0,0 +1,62 @@
+= Runtime Fabric Appliance
+
+
+== Runtime Fabric and Other PaaS Providers
+
+Anypoint Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
+
+[WARNING]
+Installing Runtime Fabric within an existing Kubernetes-based PaaS is not supported.
+
+If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
+
+== Shared Responsibility for On-Premises Runtime Fabric
```

Comment:
suggest making the type of offering explicit -- Anypoint Runtime Fabric on VMs / bare metal

Reply:
Done.